apps/src/templates/ShowCodeToggle.js

```diff
@@ -8,8 +8,6 @@ import msg from '@cdo/locale';
 import UserPreferences from '../lib/util/UserPreferences';
 import project from '../code-studio/initApp/project';
 
-const BLOCKS_GLYPH_LIGHT =
-  'data:image/gif;base64,R0lGODlhEAAQAIAAAP///////yH+GkNyZWF0ZWQgd2l0aCBHSU1QIG9uIGEgTWFjACH5BAEKAAEALAAAAAAQABAAAAIdjI+py40AowRp2molznBzB3LTIWpGGZEoda7gCxYAOw==';
```

Comment:
Are we not planning on using the light one at all any more (for example in any other themes)?

Reply:
Based on the logic from this component - no. Pane button will always be neutral white/dark button